### PR TITLE
Fix failing travis

### PR DESCRIPTION
**Pull request by martinpovolny**
This should fix failing travis on master.

ping @rvsia, @himdel 

https://travis-ci.org/ManageIQ/react-ui-components/builds/547878451?utm_medium=notification&utm_source=email

### CODE REVIEW

File: src/tagging/tests/__snapshots__/tag.modifier.test.js.snap [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Tagging modifier match snapshot 1`] = `
-<Fragment>
+<React.Fragment>
```
Comment (himdel) [inline src/tagging/tests/__snapshots__/tag.modifier.test.js.snap]: looks like this change is superfluous (and causes the new failure)

Comment (martinpovolny) [reply src/tagging/tests/__snapshots__/tag.modifier.test.js.snap]: That's that the snapshot update command did. Shall I revert it?

Comment (martinpovolny) [reply src/tagging/tests/__snapshots__/tag.modifier.test.js.snap]: It's probably about a version of something :-(

Comment (martinpovolny) [reply src/tagging/tests/__snapshots__/tag.modifier.test.js.snap]: Anyway did that.

**Pull request merged**